[2024-11-01T13:48:49.324+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:49.335+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:49.934+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:49.948+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:49.971+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:49.983+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:49.984+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:49.997+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:50.659+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:50.669+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:50.670+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:50.681+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:51.383+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:51.394+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:52.086+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:52.098+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:52.099+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:52.113+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:52.794+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:52.806+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:52.808+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:52.821+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:53.957+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:53.972+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:54.309+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:54.320+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:54.682+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:54.693+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:55.512+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:55.542+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:56.084+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:56.119+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:56.985+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:56.995+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:57.550+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:57.559+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:57.986+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:58.010+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:58.280+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:58.295+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:58.954+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:58.986+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:59.635+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:59.646+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:48:59.849+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:48:59.861+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:49:00.575+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:49:00.588+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:49:01.391+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:49:01.401+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:49:01.893+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:49:01.904+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:49:02.621+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:49:02.631+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T13:49:02.794+0000] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-01T13:49:02.807+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-11-01T14:03:33.742+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [queued]>
[2024-11-01T14:03:33.751+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [queued]>
[2024-11-01T14:03:33.752+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 3
[2024-11-01T14:03:33.777+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): get_data> on 2024-11-01 13:34:11.696229+00:00
[2024-11-01T14:03:33.783+0000] {standard_task_runner.py:60} INFO - Started process 872 to run task
[2024-11-01T14:03:33.786+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'updating_stock_data', 'get_data', 'manual__2024-11-01T13:34:11.696229+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/polars_postgres.py', '--cfg-path', '/tmp/tmphyj8ges2']
[2024-11-01T14:03:33.787+0000] {standard_task_runner.py:88} INFO - Job 162: Subtask get_data
[2024-11-01T14:03:33.847+0000] {task_command.py:423} INFO - Running <TaskInstance: updating_stock_data.get_data manual__2024-11-01T13:34:11.696229+00:00 [running]> on host 07e6c451f8a5
[2024-11-01T14:03:33.941+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow_development' AIRFLOW_CTX_DAG_ID='updating_stock_data' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-01T13:34:11.696229+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-01T13:34:11.696229+00:00'
[2024-11-01T14:03:33.945+0000] {logging_mixin.py:188} INFO - Error: Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?
[2024-11-01T14:03:33.945+0000] {python.py:202} INFO - Done. Returned value was: {}
[2024-11-01T14:03:33.975+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=updating_stock_data, task_id=get_data, execution_date=20241101T133411, start_date=20241101T140333, end_date=20241101T140333
[2024-11-01T14:03:33.999+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-11-01T14:03:34.032+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
